openselery/configuration.py

```diff
@@ -25,6 +26,9 @@ class OpenSeleryConfig(object):
         "number_payout_contributors_per_run": 1,
         "max_payout_per_run": 0.000002,
 
+        "random_split": False,
+        "full_split": False,
+
```

Comment:
Maybe better to introduce one option for all random modes:

like `random_mode: "split"` or `random_mode: "full"`. Now with boolean some combinations make no sense like `true/true`.

Reply:
true, 

there shoudl be a list of possible contribution "modes" to 

- give things names
-  remove complexity
- create transparency regarding the "possibilities" of payout

There should be no manual mode for choosing people regarding non defined metrics. And if there is such desire, it should be made obvious and clear via

 ``` random_mode: "custom" ``` or something like that

Reply:
Very good point. I will implement this suggestion.